app/wp-content/themes/kstg/library/js/vehicle.js

```diff
@@ -41,7 +53,34 @@ var calloutValue = $('.specialFeature').text();
         $('.next').prop('disabled', false);
       }
     }
+    function setEventListeners() {
+      $('.next').on('click', function() {
+        console.log(counter);
+        counter++;
+        changeActiveClassNext();
+        changeButtonState();
+        $('ul').animate({
+          left: '-=210'
+        }, 500);
+      });
 
+      $('.prev').on('click', function() {
+        counter--;
+        changeActiveClassPrev();
+        changeButtonState();
+        if (counter === 0) {
+          $(this).prop('disabled', true);
+        } else {
+          $(this).prop('disabled', false);
+        }
+        $('ul').animate({
+          left: '+=210'
+        }, 500);
+      });
+
+      //event handler for image selection
+      $('img').on('click', setActiveImage)
+    }
     $('.next').on('click', function() {
```

Comment:
What is this for? It's seems like it's doing the same thing as the handler on line 57.